### PR TITLE
fixed date and author affiliation

### PR DESCRIPTION
**Pull request by tulshi**
Version for OIDF review

### CODE REVIEW

**Review by timcappalli — APPROVED** [review]
lgtm

File: openid-risc-profile-specification-1_0.txt [hunk 1]
```diff
@@ -53,9 +53,9 @@ Table of Contents
 
 
 
-Scurtescu, et al.         Expires May 16, 2022                  [Page 1]
+Scurtescu, et al.        Expires August 8, 2022                 [Page 1]
```
Comment (FragLegs) [inline openid-risc-profile-specification-1_0.txt:56]: Can you explain how this date is chosen?

Comment (timcappalli) [reply openid-risc-profile-specification-1_0.txt:56]: +6m

**Review by FragLegs — APPROVED** [review]
Looks good!

**Pull request merged**